neurons/miners/miner.py

```diff
@@ -77,6 +78,7 @@ def get_config():
 
 
 def main(config):
+    print("Current Working Directory:", os.getcwd())
```

Comment:
use bt.logging.debug instead of print, and remove Current Working Direcottory from logging

Reply:
Addressed